src/main/resources/application.yaml

```diff
@@ -1447,10 +1447,12 @@ feature.postHearings.enabled: ${POST_HEARINGS_FEATURE:false}
 feature.postHearingsB.enabled: ${POST_HEARINGS_B_FEATURE:false}
 feature.elinksV2.enabled: false
 
-letterAsync:
-  maxAttempts: ${LETTER_ASYNC_MAX_ATTEMPTS:5}
-  delay: ${LETTER_ASYNC_DELAY:6000}
-  multiplier: ${LETTER_ASYNC_MULTIPLIER:10}
+letter-async:
+  maxAttempts: ${LETTER_ASYNC_MAX_ATTEMPTS:10}
+  delay: ${LETTER_ASYNC_DELAY:10000}
+  multiplier: ${LETTER_ASYNC_MULTIPLIER:2}
+  maxDelay: ${LETTER_ASYNC_MAX_DELAY:180000}
+  initialDelay: ${LETTER_ASYNC_INITIAL_DELAY:5000}
```

Comment:
@adusumillipraveen do we need a separate pr to change these values in flux?

Reply:
No, the defaults are sensible as per our discussion. I checked that they are not set in flux, we can override if need be.